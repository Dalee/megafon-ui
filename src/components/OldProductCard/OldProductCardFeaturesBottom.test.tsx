import * as React from 'react';
import { shallow } from 'enzyme';
import OldProductCardFeaturesBottom from './OldProductCardFeaturesBottom';
import Checked from 'icons/checked_24.svg';

const props = {
    title: 'test',
    params: [
        {
            svgIcon: <Checked />,
            title: 'test',
            caption: 'test',
            value: 'test',
        },
    ],
};

describe('<OldProductCardFeaturesBottom />', () => {
    it('it renders OldProductCardFeaturesBottom', () => {
        const wrapper = shallow(<OldProductCardFeaturesBottom {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
