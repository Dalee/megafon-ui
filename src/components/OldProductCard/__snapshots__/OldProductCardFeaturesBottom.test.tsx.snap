// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OldProductCardFeaturesBottom /> it renders OldProductCardFeaturesBottom 1`] = `
<div
  className="old-product-card-features-bottom"
>
  <Header
    as="h3"
    className="old-product-card-features-bottom__title"
    color="black"
    margin={false}
  >
    test
  </Header>
  <ul
    className="old-product-card-features-bottom__list"
  >
    <li
      className="old-product-card-features-bottom__item old-product-card-features-bottom__item_icon"
      key="test"
    >
      <test-file-stub
        className="old-product-card-features-bottom__item-icon"
      />
      <Header
        as="h5"
        className="old-product-card-features-bottom__item-title"
        color="black"
        margin={false}
      >
        test
      </Header>
      <Paragraph
        className="old-product-card-features-bottom__item-text"
        color="black"
        marginAll="none"
        sizeAll="small"
        weight="regular"
      >
        test
      </Paragraph>
    </li>
  </ul>
</div>
`;
